tests/ui/consts/value-suggestion-ice-123906.stderr

```diff
@@ -0,0 +1,18 @@
+error[E0308]: mismatched types
+  --> $DIR/value-suggestion-ice-123906.rs:3:9
+   |
+LL | fn as_chunks<const N: usize>() -> [u8; N] {
+   |                                   ------- expected `[u8; ]` because of this return type
```

Comment:
Yes, that's where I found it

Reply:
If so, these two PRs will not be able to land in the same rollup. Can you mark this as rollup=never if so?

Reply:
Well if we know that it's *going* to fail, then marking one or the other as rollup=never doesn't really do anything. That's why I unapproved the other PR, because I'd rather see this one land first.